dlt/common/normalizers/configuration.py

```diff
@@ -13,8 +13,9 @@ class NormalizersConfiguration(BaseConfiguration):
     # always in section
     __section__: str = "schema"
 
-    naming: Optional[str] = None
+    naming: Optional[Union[str, NamingConvention]] = None
```

Comment:
cool!

Reply:
maybe we should have a super simple example in the examples where we demonstrate how to implement a custom identifier normalisation with subclassing the direct naming and providing this instance or something like that.